### PR TITLE
Homework/1

### PR DESCRIPTION
**Pull request by canzoroglu**

### CODE REVIEW

File: BMICalculator/Program.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+                int age = int.Parse(Console.ReadLine());
+                Console.Write("Enter your weight in kg: ");
+                double weight = double.Parse(Console.ReadLine());
+                Console.Write("Enter your height in cm: ");
+                double heightC = double.Parse(Console.ReadLine());
+                double heightM = heightC / 100.0;
+                string msg = $"{name} {surname} is {age} years old, his weight is {weight} kg and his height is {heightC} cm.";
+                double bmi = weight / (heightM * heightM);
+                Console.WriteLine(msg);
+                Console.WriteLine($"{name}'s BMI is {bmi:F2}");
```
Comment (Almantask) [inline BMICalculator/Program.cs]: Split this into functions

File: BMICalculator/Program.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    {
+        static void Main(string[] args)
+        {
+        	// To be able to use "." instead of "," as decimal separator
+            System.Globalization.CultureInfo customCulture = (System.Globalization.CultureInfo)System.Threading.Thread.CurrentThread.CurrentCulture.Clone();
+            customCulture.NumberFormat.NumberDecimalSeparator = ".";
+            System.Threading.Thread.CurrentThread.CurrentCulture = customCulture;
+
+            for (int i = 0; i < 2; i++)
+            {
```
Comment (Almantask) [inline BMICalculator/Program.cs]: A single iteration could be a function call and just call that function twice. Loop is a little overkill.

**Review by Almantask — CHANGES_REQUESTED** [review]

File: BMICalculator/Program.cs (hunk 2 above)
Comment (canzoroglu) [reply BMICalculator/Program.cs]: Thank you for the review. I've updated the code.

**Review by Almantask — APPROVED** [review]
Looks better now. Though it's too early for OOP.

**Pull request closed without merging**